changelogs/client_server/newsfragments/1530.new

```diff
@@ -0,0 +1 @@
+[`POST /_matrix/client/v3/login/get_token`](/client-server-api/#post_matrixclientv3loginget_token).
```

Comment:
I wasn't aware of this type of newsfragment being a thing. We have other new endpoints (`POST /_matrix/media/v1/create` etc) which haven't had one of these...

Reply:
Yea... They'll be caught as part of "changelog normalization" in the release process.